Fixed an issue where sentence end with period validation failed in case of multi-lines descriptions. (#3595)
demisto/demisto-sdk: CHANGELOG.md

# Changelog
## Unreleased
* Fixed an issue where the **coverage-analyze** command was not parsing the logs correctly.

> * Fixed an issue where sentence end with period validation failed in case of multi-lines descriptions.

Reviewer: ```suggestion
* Fixed an issue where sentence end with period validation failed when descriptions ended with a newline.
```
